src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1771,6 +1780,45 @@ bool OutputStateMachineEngine::_GetOscSetColorTable(const std::wstring_view stri
     return success;
 }
 
+// Routine Description:
+// - Given a hyperlink string, attempts to parse the URI encoded. An 'id' parameter
+//   may be provided.
+//   If there is a URI, the well formatted string looks like:
+//          "<params>;<URI>"
+//   If there is no URI, we need to close the hyperlink and the string looks like:
+//          ";"
+// Arguments:
+// - string - the string containing the parameters and URI
+// - params - where to store the parameters
+// - uri - where to store the uri
+// Return Value:
+// - True if a URI was successfully parsed or if we are meant to close a hyperlink
+bool OutputStateMachineEngine::_ParseHyperlink(const std::wstring_view string,
+                                               std::wstring& params,
+                                               std::wstring& uri) const

```

Comment:
```suggestion
                                               const std::wstring& params,
                                               const std::wstring& uri) const
```

Reply:
But we're assigning a new value to `uri` and `params` (after parsing the string) - adding const there gives an error

Reply:
derp, that's my bad.